frontend/pages/hosts/ManageHostsPage/ManageHostsPage.jsx

```diff
@@ -31,6 +31,8 @@ import {
 import PATHS from "router/paths";
 import deepDifference from "utilities/deep_difference";
 
+import hostActions2 from "services/entities/hosts";
```

Comment:
Perhaps `hostClient` or something like that? I think it would be helpful to remove the term "action" since that had a specific meaning when used with Redux.